src/main/java/bio/terra/service/profile/google/GoogleBillingService.java

```diff
@@ -43,7 +43,7 @@ private static CloudBillingClient accessClient(AuthenticatedUserRequest user) {
       // See https://developers.google.com/identity/protocols/application-default-credentials
       // for more information.
       List<String> scopes =
-          Collections.singletonList("https://www.googleapis.com/auth/cloud-platform");
+          Collections.singletonList("https://www.googleapis.com/auth/cloud-billing.readonly");
       if (serviceAccountCredentials.createScopedRequired()) {
```

Comment:
There are other cases in `GoogleProjectService` where the code checks `createScopeRequired()` and then uses `https://www.googleapis.com/auth/cloud-platform`. Do those scope not need to be narrowed as well?

Reply:
In that case, those are scope that we can explicitly ask for from the service account so I'm not as worried there